src/utils/translation/translate.ts

```diff
@@ -0,0 +1,55 @@
+// To use, remove lang attribute from index.html, import translate function, and call with translation key:
+// translate("~INTRO.HELLO")
```

Comment:
Conventional usage should look like this:
```typescript
import t from "utils/translate";

console.log(t("~INTRO.HELLO"));
```
The call to the translate function should be as inconspicuous as possible.

Reply:
Yeah, that's how I was using it in practice, but you're right that this comment makes it unclear.  I'll clarify so it makes more sense.

Reply:
Comment could still use some improving...

Reply:
Fixed.